bundler/src/lib.rs

```diff
@@ -112,6 +125,14 @@ impl Bundler {
     }
 }
 
+#[derive(Serialize_tuple, Deserialize_tuple, Clone)]
+pub struct Manifest {
+    pub version: u32,
+    pub data: Cid,
```

Comment:
If you can just document the available versions and how the data should be interpreted depending on them, it would be ace.

Reply:
Of course! will do.